data.php

```diff
@@ -33,10 +33,11 @@
 
     $picture_input = $_POST["picture"];
 
-     if (strpos($_POST["what_to_order"], 'carling') !== false){
+    $carling_query = strtolower($_POST['what_to_order']);
+    if (strpos($carling_query, 'carling') !== false) {
```

Comment:
I suggest creating some kind of Carling validator function that you can reuse!

Reply:
While I really like this functionality, modern best practice would actually be to redirect any user that attempts to enter 'carling' to a special PHP file called `get_out_of_my_pub.php` - the purpose of this special file is to use Peggy Mitchell to humiliate and shame the user into rethinking their life choices.

This ultra-modern design pattern is often referred to as the 'anti-carling response' and is pretty widely considered to be the best option for dealing with this classic programming problem.

Reply:
Really great to know what is industry standard in this regard, thanks